aws/cloudformation-templates/template.yaml

```diff
@@ -239,6 +254,33 @@ Parameters:
       - 'No'
     Default: 'Yes'
 
+  AmazonPayPublicKeyId:
+    Type: String
+    Description: >
+      This demonstration store contains an example of signing Amazon Pay payloads for integration with an Amazon Pay sandbox.
+      This parameter is the Public Key ID for the Amazon Pay sandbox account being used.
+      For more information see "Amazon Pay Integration" under the README file in the amazon-pay-signing subfolder in the repository.
+
+  AmazonPayStoreId:
+    Type: String
+    Description: >
+    The Store ID of the Amazon Pay sandbox account being used.
+    For more information see "Amazon Pay Integration" under the README file in the amazon-pay-signing subfolder in the repository.
+
+  AmazonPayMerchantId:
+    Type: String
+    Description: >
+      Merchant ID of the Amazon Pay sandbox account being used.
+      For more information see "Amazon Pay Integration" under the README file in the amazon-pay-signing subfolder in the repository.
+
+  AmazonPayPrivateKey:
+    Type: String
+    Description: >
+      The private key to be used for signing Amazon Pay requests for a sandbox account.
+      You will need to copy the whole contents of the .pem file into this parameter.
+      For more information see "Amazon Pay Integration" under the README file in the amazon-pay-signing subfolder in the repository.
+    NoEcho: true
+
```

Comment:
Sorry I missed this earlier. These parameters all need suitable defaults in the workshop template [base-workshop.yaml](https://github.com/aws-samples/retail-demo-store/blob/master/aws/cloudformation-templates/event-engine/base-workshop.yaml) such that the project deploys and operates cleanly without Amazon Pay configured. The workshop template is used to launch 1:many workshop experiences.

Reply:
OK this is done now, and we have tested as far as initiating deployment with the workshop template. Hopefully next time I will remember to do this before prompting! 

I have also taken the liberty of adding one commit making whether or not Amazon Pay is enabled more declarative in the UI (`Checkout.vue`):

    <amazon-pay-button v-if="placeOrderEnabled && amazonPayEnabled" @click.native="finalizeAmazonPayOrder"/>

...

    amazonPayEnabled() {
      const enabled = process.env.VUE_APP_AMAZON_PAY_PUBLIC_KEY_ID && process.env.VUE_APP_AMAZON_PAY_PUBLIC_KEY_ID !== '' &&
                      process.env.VUE_APP_AMAZON_PAY_STORE_ID && process.env.VUE_APP_AMAZON_PAY_STORE_ID !== '' &&
                      process.env.VUE_APP_AMAZON_PAY_MERCHANT_ID && process.env.VUE_APP_AMAZON_PAY_MERCHANT_ID !== ''
      if(enabled) {
        console.log('Amazon Pay Button Enabled')
      }
      return enabled;
    }